src/query/api/v1/middleware/rewrite_test.go

```diff
@@ -214,12 +219,71 @@ func TestPrometheusRangeRewrite(t *testing.T) {
 			expectedQuery:    "foo",
 			expectedLookback: durationPtr(15 * time.Minute),
 		},
+		{
+			name: "instant query; rewrite w/ prom engine",
+			attrs: []storagemetadata.Attributes{
+				{
+					MetricsType: storagemetadata.AggregatedMetricsType,
+					Resolution:  5 * time.Minute,
+					Retention:   90 * 24 * time.Hour,
+				},
+			},
```

Comment:
Nit: there was some effort in https://github.com/m3db/m3/pull/3882 to tidy up those test tables. Kindly please use a helper function to build these `attrs` as a oneliner.